lib/progressTracker.js

```diff
@@ -63,18 +62,32 @@ function track (instance, opts) {
 
     addedListeners = true
 
+    function cancel () {
+      if (iOpts.amount) {
+        amountProgressBar.tick(iOpts.amount - 1)
+      } else {
+        durationProgressBar.tick(iOpts.duration - 1)
+      }
+      instance.cancel()
+    }
+
     // note: Attempted to curry the functions below, but that breaks the functionality
     // as they use the scope/closure of the progress bar variables to allow them to be reset
     if (opts.outputStream.isTTY) {
+      process.once('SIGINT', cancel)
```

Comment:
I would prefer not having a SIGINT handler in this file at all.

Reply:
Make sense, I will do the change